langchain/tools/image/tool.py

```diff
@@ -0,0 +1,109 @@
+"""This tool allows agents to generate images.
+
+To use this tool, you must first set as environment variables:
+    STEAMSHIP_API_KEY
+```
+"""
+from enum import Enum
+from typing import Optional, Dict
+
+from pydantic import root_validator
+from steamship import Steamship
```

Comment:
Since we don't want `Steamship` as a strict dependency, we should import only for type annotations and forward ref resolution in the pydantic class. Something ugly like:

```suggestion
if TYPE_CHECKING:
    from steamship import Steamship
else:
    try:
        from steamship import Steamship
    except ImportError:
         pass
```

Reply:
Oops. Didn't think of that (it's my first contribution). Will keep that in mind for future PRs.